### DISCUSSION TITLE
cant install features (19,22,23) from installer ofer ssh

### DISCUSSION THREAD

**Discussion by SammyD1986** [Guilouz/Creality-Helper-Script-Wiki#563] [Issues]
I cant install the OctoEverywhere or other features over the ssh install.
The features are number 19,22,23  if i choose one of them the installer for simplyprint start.

Why do you do this in the installermenu:

 #run "install_octoeverywhere" "install_menu_ui_k1"
          run "install_simplyprint" "install_menu_ui_k1"

 #run "install_octoapp_companion" "install_menu_ui_k1"
          run "install_simplyprint" "install_menu_ui_k1"

**Comment by leonettii** [comment]
I believe there is currently an issue where these are all failing to install the correct dependencies so their installers have been temporarily disabled with the recent update, if you scroll up after the installer skips those options you will see the line "✗ Due to some compatibility issues, this feature is temporarily disabled!"

**Comment by Guilouz** [comment]
It's now fixed.
